tests/integration/cli/test_cli.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import importlib
+import inspect
+from pathlib import Path
+
+import pytest
+
+from tests.integration.cli.test_helper import check_run
+
+# This assumes have OTX installed in environment.
+otx_module = importlib.import_module("otx")
+RECIPE_PATH = Path(inspect.getfile(otx_module)).parent / "recipe"
+RECIPE_LIST = [str(_.relative_to(RECIPE_PATH)) for _ in RECIPE_PATH.glob("**/*.yaml")]
+
+# [TODO]: This is a temporary approach.
+DATASET = {
+    "classification": {
+        "data_dir": "tests/assets/classification_dataset",
+        "overrides": [
+            "model.otx_model.config.head.num_classes=2",
+            "model.otx_model.config.data_preprocessor.num_classes=2",
+        ],
+    },
+    "detection": {
+        "data_dir": "tests/assets/car_tree_bug",
+        "overrides": ["model.otx_model.config.bbox_head.num_classes=3"],
+    },
+}
+
+
+@pytest.mark.parametrize("recipe", RECIPE_LIST)
+def test_otx_train(recipe: str, tmp_path: Path) -> None:
+    """
+    Test the 'otx train' command with 1 epochs trainig and check outputs.
+
+    Args:
+        recipe (str): The recipe to use for training. (eg. 'classification/otx_mobilenet_v3_large.yaml')
+        tmp_path (Path): The temporary path for storing the training outputs.
+
+    Returns:
+        None
+    """
+    task = recipe.split("/")[0]
+    model_name = recipe.split("/")[1].split(".")[0]
+    tmp_path = tmp_path / f"otx_train_{model_name}"
+    command_line = [
+        "otx", "train",
+        f"+recipe={recipe}",
+        f"base.data_dir={DATASET[task]['data_dir']}",
+        "trainer.max_epochs=1",
+        "trainer.check_val_every_n_epoch=1",
+        "trainer=auto",
```

Comment:
Rather than manually overriding at the variable level, consider to override this https://github.com/openvinotoolkit/training_extensions/blob/v2/src/otx/config/debug/default.yaml

Reply:
Thanks for the comment. I will create a PR to reflect your comments. I have a question regarding this comment. I want to create a config file called `debug/intg_test.yaml` and want to apply it to test. 
Can you tell me what args should be added to override the config in this debug folder in the cli?

Reply:
I guess that you need to add `debug` optional field to `TrainConfig` as same as `recipe`.
<img width="229" alt="image" src="https://github.com/openvinotoolkit/training_extensions/assets/26541465/4c6685bc-0d36-4e02-85b1-374d9637f153">
Then, it might be working by adding argument `+debug=intg_test`. This is exactly same with what we do with `+recipe=???`.

Reply:
Thank you. I'll give it a try.